src/helpers/issue-fetching.ts

```diff
@@ -296,3 +318,41 @@ function castCommentsToSimplifiedComments(comments: Comments, params: FetchParam
       url: comment.html_url,
     }));
 }
+
+export async function fetchLinkedPrFromIssue(owner: string, repo: string, issueNumber: number, context: Context) {
+  const prs = await context.octokit.rest.pulls.list({
```

Comment:
Might be best to wrap this in a try-catch because if e.g the repo has been deleted this will likely throw